test/behavior/asm.zig

```diff
@@ -36,7 +35,6 @@ test "output constraint modifiers" {
     if (builtin.zig_backend == .stage2_arm) return error.SkipZigTest; // TODO
     if (builtin.zig_backend == .stage2_aarch64) return error.SkipZigTest; // TODO
     if (builtin.zig_backend == .stage2_sparc64) return error.SkipZigTest; // TODO
-    if (builtin.zig_backend == .stage2_c) return error.SkipZigTest; // TODO
```

Comment:
I disabled these because x64 MSVC doens't support *any* inline asm, and x86 MSVC doesn't support the asm syntax zig uses. Andrew proposed creating a "C" dialect for use in these cases, but I don't think that got formalized.

Reply:
I still want test coverage for this on some target, so I'll just disable them on windows.